prompts/sources.go

```diff
@@ -148,13 +148,55 @@ func sourceBaseForm(quickstart *Quickstart) (*QuickstartState, error) {
 
 func AddToSource(name string, currentSource *workflow.Source) (*workflow.Source, error) {
 	addOpenAPIFile := false
+	selectedDoc := "new"
 	if _, err := tea.NewProgram(charm_internal.NewForm(huh.NewForm(
-		charm_internal.NewBranchPrompt("Would you like to add an openapi file to this source?", &addOpenAPIFile)),
-		fmt.Sprintf("Let's add to the source %s", name))).
+		huh.NewGroup(
+			huh.NewSelect[string]().
+				Title("Would you like to modify an existing OpenAPI document or add a new one?").
```

Comment:
I feel like this implies you are changing the contents of the doc rather than the config around it

Reply:
Any suggested prompt?

Reply:
Would you like to modify the settings of an existing OpenAPI document or add a new one?

Reply:
That works for me. Or if the only setting is `location` we could just say `... modify the location of...`